packages/react-scripts/package.json

```diff
@@ -55,31 +55,31 @@
     "html-webpack-plugin": "4.0.0-beta.5",
     "identity-obj-proxy": "3.0.0",
     "is-wsl": "^1.1.0",
-    "jest": "24.7.1",
+    "jest": "24.8.0",
     "jest-environment-jsdom-fourteen": "0.1.0",
-    "jest-resolve": "24.7.1",
-    "jest-watch-typeahead": "0.3.0",
+    "jest-resolve": "24.8.0",
+    "jest-watch-typeahead": "0.3.1",
     "mini-css-extract-plugin": "0.5.0",
-    "optimize-css-assets-webpack-plugin": "5.0.1",
-    "pnp-webpack-plugin": "1.2.1",
+    "optimize-css-assets-webpack-plugin": "5.0.3",
+    "pnp-webpack-plugin": "1.5.0",
     "postcss-flexbugs-fixes": "4.1.0",
     "postcss-loader": "3.0.0",
     "postcss-normalize": "7.0.1",
-    "postcss-preset-env": "6.6.0",
+    "postcss-preset-env": "6.7.0",
     "postcss-safe-parser": "4.0.1",
     "react-app-polyfill": "^1.0.1",
     "react-dev-utils": "^9.0.1",
-    "resolve": "1.10.0",
+    "resolve": "1.12.0",
     "sass-loader": "7.1.0",
-    "semver": "6.0.0",
+    "semver": "6.3.0",
     "style-loader": "0.23.1",
```

Comment:
1.0 came out, it fixes sass sourcemaps.

Reply:
Bumped to `1.0.0`, however we only enable source maps for CSS in production, so the source maps won't be used by `style-loader` which is only used in dev.